CorsixTH/Lua/languages/english.lua

```diff
@@ -268,6 +268,8 @@ adviser = {
     high_prices = "Your charges for %s are high. This will make big profits in the short-term, but ultimately you'll start to drive people away.",
     fair_prices = "Your charges for %s seem fair and balanced.",
     patient_not_paying = "A patient left without paying for %s because it's too expensive!",
+    no_doctor_no_gp_office = "You should build a GP's Office and hire a doctor at some point!",
+    no_gp_office = "You haven't built a GP's Office where your doctors can diagnose the patients!",
```

Comment:
You could change the the to `the` to `their` or drop the word entirely. But now it's nits on nits so I'd approve this as is anyway.